Source/Extensions/Blazorise.DataGrid/_DataGridModal.razor

```diff
@@ -31,7 +31,7 @@
                     <Fields>
                         @foreach ( var column in OrderedEditableColumns )
                         {
-                            <Field ColumnSize="@column.PopupFieldColumnSize" Padding="Blazorise.Padding.Is2.OnX.Is2.OnY">
+                            <Field ColumnSize="@(column.EditFieldColumnSize ?? column.PopupFieldColumnSize)" Padding="Blazorise.Padding.Is2.OnX.Is2.OnY">
```

Comment:
This might not quite result in correct reverse compatibility. Better do some sync logic, I believe we need an internal field for that. Since both Edit and Popup have defined values, it's currently a breaking change for people that are currently using `PopupFieldColumnSize`

Reply:
Or just introduce `EditFieldColumnSize` as  null so the coalescing operators act ok.

Reply:
How did I not notice the default values?! 🤦‍♂️

I have removed them, and I think the logic is now OK.